src/rules/wf/test.rs

```diff
@@ -217,7 +217,7 @@ fn projection_type_in_header() {
 
             // Projection types in an impl header are not assumed to be well-formed,
             // an explicit where clause is needed (see below).
```

Comment:
The comment is obsolete and can be removed.

Reply:
I think it's still mostly correct, but I changed the wording some.